src/Components/ContentDelivery/ContentDelivery.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ContentDelivery from './ContentDelivery.js'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { BrowserRouter } from 'react-router-dom'
+import { VideoProvider } from '../../Context/VideoContext'
+
+describe('ContentDelivery component', () => {
+  it('should display a video player and skip button when rendered', () => {
+
+    render(
+      <BrowserRouter>
+        <VideoProvider>
+          <ContentDelivery/>
+        </VideoProvider>
+      </BrowserRouter>
+    )
+
+    const breakTitle = screen.getByRole('heading', { name: 'Enjoy Your Break!'})
+    const skipBtn = screen.getByRole('button', { name: 'Skip video' })
+
```

Comment:
I've been really digging destructuring these RTL methods like:
`const { getByRole } = render(...)` 
So that way we don't need to use `screen.` but I get if that's not everybody's vibe! Twould be great to establish company convention on this.

Reply:
Sure, we can chat about that. I tend to prefer using screen only because I think it's slightly more readable, but definitely open to either way. I'd need a reminder about syntax though!